src/BugsnagUnity/IConfiguration.cs

```diff
@@ -49,7 +49,7 @@ public interface IConfiguration
 
         string Context { get; set; }
 
-        LogType NotifyLevel { get; set; }
+        LogType NotifyLogLevel { get; set; }
```

Comment:
Is this a breaking change for anyone initializing bugsnag-unity via code rather than game object? I would have imagined both would need to be defined on the interface for `NotifyLevel` to remain accessible.

Reply:
NotifyLevel is still there in the public configuration class (and has a warning on it now). 

The interface is only referenced behind the scenes in the client etc